src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -69,7 +69,9 @@ private static ImmutableArray<string> GetTokenTypes()
     {
         var builder = ImmutableArray.CreateBuilder<string>();
 
+#pragma warning disable CS0618 // RoslynTokenTypes is obsolete
```

Comment:
Do we need an issue to follow up on this?

Reply:
I was actually going to comment that we should just fix this now, in this PR, but at the very least a follow up issue for sure.

Reply:
:+1:

Reply:
Created follow up issue https://github.com/dotnet/razor/issues/8791